docs/dense-retrieval.md

```diff
@@ -41,7 +41,8 @@ recall_1000             all     0.9618
 ```
 
 > To evaluate with on-the-fly query encoding, replace `--encoded-queries` with our pretrained encoder model
-> `--encoder tct_colbert-msmarco`. The encoding will run on CPU by default. To enable GPU, add `--device cuda:0`
+> on [Hugging Face](https://huggingface.co/castorini/tct_colbert-msmarco/tree/main) 
+> `--encoder castorini/tct_colbert-msmarco`. The encoding will run on CPU by default. To enable GPU, add `--device cuda:0`.
```

Comment:
Why you're at it, remove the '>'? Dunno why this part of the documentation has that?